src/applications/hca/config/chapters/householdInformation/dependentInformation.js

```diff
@@ -0,0 +1,41 @@
+import { assign } from 'lodash/fp';
+import fullSchemaHca from 'vets-json-schema/dist/10-10EZ-schema.json';
+
+import DependentView from '../../../components/DependentView';
+import { uiSchema as dependentUI } from '../../../definitions/dependent';
+
+const { dependents } = fullSchemaHca.properties;
+
+export default {
+  path: 'household-information/dependent-information',
+  title: 'Dependent information',
+  depends: data => data.discloseFinancialInformation,
+  uiSchema: {
+    'view:reportDependents': {
+      'ui:title': 'Do you have any dependents to report?',
+      'ui:widget': 'yesNo',
+    },
+    dependents: {
+      items: dependentUI,
+      'ui:options': {
+        expandUnder: 'view:reportDependents',
+        itemName: 'Dependent',
+        hideTitle: true,
+        viewField: DependentView,
+      },
+      'ui:errorMessages': {
+        minItems: 'You must add at least one dependent.',
```

Comment:
I couldn't find an exact reference, but "must" is too strong of a language to use for feedback... the other feedback messaging examples](https://design.va.gov/patterns/messaging-dictionary) start with "please"... I'd get feedback on this from your content specialist.

Reply:
Thanks for pointing this out I know @jonathanjnelson is planning on doing a overhaul of the content on this form... Will get with him and open another `tech debt` ticket to overview all error handling messages across the form.